### DISCUSSION TITLE
How should I expect a helm release to update if a values from secret is updated

### DISCUSSION THREAD

**Discussion by punkle** [fluxcd/flux2#1234] [Q&A] [answered]
Hi, I have a helm release with valuesFrom a secret a bit like this:

  valuesFrom:
    - kind: Secret
      name: mysecret
      valuesKey: values.yaml
      optional: true
      namespace: default
      
I have updated the secret but I am not seeing that reflected in the helmrelease. Should/can I expect that to update?

**Comment by stefanprodan** [comment, marked as answer]
The secret is refreshed at every `HelmRelease.spec.interval`.

**Reply by hiddeco** [reply]
Or if you manually reconcile the release with `flux reconcile helmrelease <name>`.
